jabgui/src/main/java/org/jabref/gui/maintable/RightClickMenu.java

```diff
@@ -171,11 +172,14 @@ private static Menu createCopySubMenu(ActionFactory factory,
                                           DialogService dialogService,
                                           StateManager stateManager,
                                           GuiPreferences preferences,
+                                          LibraryTab libraryTab,
                                           ClipBoardManager clipBoardManager,
                                           JournalAbbreviationRepository abbreviationRepository,
                                           TaskExecutor taskExecutor) {
         Menu copySpecialMenu = factory.createMenu(StandardActions.COPY_MORE);
 
+        clipBoardManager.setSourceBibDatabaseContext(libraryTab.getBibDatabaseContext());
```

Comment:
This is toatally wrong.

Every thought of switchting tabs? Ever thought about the order of calls? Ever thoght about that there are key bindings such as <kbd>Ctrl</kbd>+<kbd>C</kbd>?

--> move this to `org.jabref.gui.edit.EditAction#execute` at the correct place.